### PR TITLE
Delete duplicate examples from reorganization

### LINKED ISSUE #100: organize examples
we need to re-organize the examples in `framework/example` name

before merging, we need to let the engineering team know since the platform depends on some of those links

### PR DESCRIPTION
**Pull request by bryannho**
Changes:
- Deletes duplicate example apps which have been moved to follow the pattern `framework/example-app`
- Once the frontend PR is merged, we can merge this

Related PRs:
- https://github.com/ploomber/doc/pull/126
- https://github.com/ploomber/cloud-frontend/pull/370 

Fixes #100 

----
📚 Documentation preview 📚: https://ploomber-doc--130.org.readthedocs.build/en/130/

### CODE REVIEW

**Comment by edublancas** [comment]
@bryannho please ping me once the frontend PR is merged and deployed

**Review by edublancas — APPROVED** [review]

**Comment by bryannho** [comment]
@edublancas FIxed merge conflicts. Needs another review but should be good to merge

**Review by edublancas — APPROVED** [review]

**Pull request merged**
